_posts/en/newsletters/2025-12-19-newsletter.md

```diff
@@ -0,0 +1,656 @@
+
+  `OP_CHECKCONTRACTVERIFY` (CCV) [became][ccv bip] [BIP443][] with
+  [refined][ccv semantics] semantics especially around flow of funds. CCV
+  enables reactive security vaults, and a wide array of other contracts by
+  constraining the `scriptPubKey` and amount of an input or output in certain
+  ways. The `OP_VAULT` proposal was [withdrawn][vault withdrawn] in favor of
+  CCV as well. For more on CCV's applications, see [MATT][topic MATT].
+
+  A set of 64-bit arithmetic opcodes were [proposed][64bit bip]. Bitcoin's
+  current math operations are (surprisingly) not able to operate on the full
+  range of bitcoin input and output amounts. Combined with other opcodes to
+  access and/or constrain input/output amounts these expanded arithmetic
+  operations could enable new bitcoin wallet functionality.
+
+  [OP_TXHASH][txhash] got a [variant][txhash sponsors] that would enable
+  [transaction sponsorship][topic fee sponsorship].
+
+  Developers proposed two options for giving Script elliptic curve
+  cryptographic operations other than CHECKSIG and friends. One
+  [proposes][tweakadd] `OP_TWEAKADD` to enable constructing taproot
+  `scriptPubKeys`. The other [proposes][ecmath] more granular elliptic curve
+  opcodes such as `EC_POINT_ADD` motivated by similar functionality, but with
+  more applications such as new signature verifications or multi-signature
+  functionality. Either of these proposals could be combined with `OP_TXHASH`
+  and 64-bit arithmetic (or similar opcodes) to enable functionality similar
+  to CCV.
+
+{:scriptrestoration}
+- **Script Restoration:** A series of 4 BIPs were [posted][gsr bips] for the
+  Script Restoration project. The Script changes and opcodes proposed in these
+  4 BIPs would enable all of the functionality proposed in the above opcode
+  proposals and more while restoring and updating Script to be more useful to
+  developers and users alike.
+
+</div>
+
+## July
+
+{:#ccdelegation}
+- **Chain code delegation:** ...
+
+## August
+
+{:#utreexo}
+- **Utreexo draft BIPs:** ...
+
+{:#minfeerate}
+- **Lowering the minimum relay feerate:** ...
+
+{:#templatesharing}
+- **Peer block template sharing:** ...
+
+{:#fuzzing}
+- **Differential fuzzing of Bitcoin and LN implementations:** ...
+
+## September
+
+{:#simplicity}
+- **Details about the design of Simplicity:** ...
+
+{:#eclipseattacks}
+- **Partitioning and eclipse attacks using BGP interception:** ...
+
+<div markdown="1" class="callout" id="releases">
+
+## Summary 2025: Major releases of popular infrastructure projects
+
+FIXME:Gustavojfe
+
+</div>
+
+## October
+
+{:#arbdata}
+- **Theoretical limitations on embedding data in the UTXO set:** ...
+
+{:#channeljamming}
+- **Channel jamming mitigation simulation results and updates:** Carla
+  Kirck-Cohen in collaboration with Clara Shikhelman and elnosh had updated the
+  [lightning jamming simulation results][channel jamming results], and updates
```

Comment:
```suggestion
  [Lightning jamming simulation results][channel jamming results], and updates
```

Reply:
updated in [63b9e37](https://github.com/bitcoinops/bitcoinops.github.io/pull/2574/commits/63b9e37b270190496b99306c8704de68dba5ea23)